doc/script_commands.txt

```diff
@@ -8174,6 +8174,18 @@ Opens the enchantgrade UI for the attached character or the player given by the
 
 This command requires packet version 2020-07-24 or newer.
 
+---------------------------------------
+
+*set_reputation_points(<type>,<points>{,<char id>})
+
+Sets the reputation points <points> for reputation group <type> for the attached player or the given character id.
```

Comment:
```suggestion
Sets the reputation points via <points> for reputation group <type> for the attached player or the given character ID.
```

Reply:
Can you add a sentence to say that the `<type>` is the "Client side index"? the expected parameter is not clear enough